assembly/std/context.ts

```diff
@@ -71,7 +67,7 @@ export function addressStack(): Array<Address> {
  * Caller is the person or the smart contract that directly called
  * the pending function.
  *
- * @return {Address} returns the stack element just below the top of the address stack
+ * @returns the stack element just below the top of the address stack
```

Comment:
Man, that comment about the returned value is scary. You should either delete it or just say it returns the caller's address.